dist/react-simpletabs.js

```diff
@@ -75,71 +75,68 @@ return /******/ (function(modules) { // webpackBootstrap
 	var Tabs = React.createClass({
 	  displayName: 'Tabs',
 	  propTypes: {
-	    className: React.PropTypes.oneOfType([
-	      React.PropTypes.array,
-	      React.PropTypes.string,
-	      React.PropTypes.object
-	    ]),
+	    className: React.PropTypes.oneOfType([React.PropTypes.array, React.PropTypes.string, React.PropTypes.object]),
 	    tabActive: React.PropTypes.number,
 	    onMount: React.PropTypes.func,
 	    onBeforeChange: React.PropTypes.func,
 	    onAfterChange: React.PropTypes.func,
-	    children: React.PropTypes.oneOfType([
-	      React.PropTypes.array,
-	      React.PropTypes.element
-	    ]).isRequired
+	    children: React.PropTypes.oneOfType([React.PropTypes.array, React.PropTypes.element]).isRequired
 	  },
-	  getDefaultProps:function () {
+	  getDefaultProps: function getDefaultProps() {
```

Comment:
More a question than a comment:

What is the difference between `getDefaultProps:function () {` and `getDefaultProps: function getDefaultProps() {`